### PR TITLE
Added file changes API documentation

### PR DESCRIPTION
**Pull request by aleksandrychev**
ChangeLog: Title
Ticket: ENT-4734

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a PR! Maybe @craigcomstock can review this?

**Comment by aleksandrychev** [comment]
@craigcomstock please review

File: reference/enterprise-api-ref/file-changes.markdown [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+## File changes statistics
+
+**URI:** https://hub.cfengine.com/api/file-changes/statistics?fromTime=:fromTime&toTime=:toTime
+
+**Method:** GET
+
+Get file changes statistics by period.
+
+* **fromTime** *(timestamp)*
+    Include changes performed within interval.
```
Comment (craigcomstock) [inline reference/enterprise-api-ref/file-changes.markdown]: I see from example below what the format of (timestamp) is but I think it should be explained here. Something like YYYY-mm-dd%HHMM:SS maybe? (using the letters from `man date`).

File: reference/enterprise-api-ref/file-changes.markdown [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    "data": {
+        "DIFF": {
+            "20 Sep 2019": 0,
+            "21 Sep 2019": 0,
+            "22 Sep 2019": 0,
+            "23 Sep 2019": 0,
+            "24 Sep 2019": 0,
+            "25 Sep 2019": 0,
+            "26 Sep 2019": 0,
+            "27 Sep 2019": 23
```
Comment (craigcomstock) [inline reference/enterprise-api-ref/file-changes.markdown]: what do the numbers 0, 23, 33 mean as values of the various dates in the assoc array? Would be good to explain the structure of the response.

Comment (aleksandrychev) [reply reference/enterprise-api-ref/file-changes.markdown]: added

File: reference/enterprise-api-ref/file-changes.markdown (hunk 1 above)
Comment (aleksandrychev) [reply reference/enterprise-api-ref/file-changes.markdown]: added

**Comment by craigcomstock** [comment]
@aleksandrychev looks great. squash commits, maybe move ticket from title to a line by itself: Ticket: ENT-4734.

**Comment by craigcomstock** [comment]
@aleksandrychev looks like you might need to "sign the CLA" for documentation repo and squash the "review changes" commit. :+1:

**Comment by aleksandrychev** [comment]
@craigcomstock done

**Comment by aleksandrychev** [comment]
>one more thing, I think the commit needs to the Ticket: ENT-4734 in it.

@craigcomstock , done

**Pull request merged**